src/phone-input/__tests__/phone-input.e2e.js

```diff
@@ -34,7 +34,14 @@ describe('PhoneInput', () => {
   });
 
   it('passes basic a11y tests', async () => {
-    const accessibilityReport = await analyzeAccessibility(page);
+    const accessibilityReport = await analyzeAccessibility(page, {
```

Comment:
I am not sure why it started failing - does anyone have any ideas? 🤔

Reply:
can you please share the axe error or remove the rule option so I can view it in CI?

Reply:
it was complaining because of this: https://dequeuniversity.com/rules/axe/3.0/label

Reply:
I believe you can add an aria-label to the `Input` through overrides